server/README.md

```diff
@@ -42,6 +42,7 @@ To run the server locally, do the following:
 Development requires you having Node.js v4 or later installed.  
 To run server for development just follow instuctions from [Using Node](#using-node) section.  
 To run test suite simply execute `npm test`.
+To see all available commands, see package.json.
```

Comment:
I also added a line, pointing people towards `package.json` to see all available commands.